go/vt/orchestrator/db/generate_base.go

```diff
@@ -28,21 +28,21 @@ var generateSQLBase = []string{
           version varchar(128) CHARACTER SET ascii NOT NULL,
           binlog_format varchar(16) CHARACTER SET ascii NOT NULL,
           log_bin tinyint(3) unsigned NOT NULL,
-          log_slave_updates tinyint(3) unsigned NOT NULL,
+          log_replica_updates tinyint(3) unsigned NOT NULL,
           binary_log_file varchar(128) CHARACTER SET ascii NOT NULL,
           binary_log_pos bigint(20) unsigned NOT NULL,
-          master_host varchar(128) CHARACTER SET ascii NOT NULL,
-          master_port smallint(5) unsigned NOT NULL,
-          slave_sql_running tinyint(3) unsigned NOT NULL,
-          slave_io_running tinyint(3) unsigned NOT NULL,
-          master_log_file varchar(128) CHARACTER SET ascii NOT NULL,
-          read_master_log_pos bigint(20) unsigned NOT NULL,
-          relay_master_log_file varchar(128) CHARACTER SET ascii NOT NULL,
-          exec_master_log_pos bigint(20) unsigned NOT NULL,
-          seconds_behind_master bigint(20) unsigned DEFAULT NULL,
-          slave_lag_seconds bigint(20) unsigned DEFAULT NULL,
-          num_slave_hosts int(10) unsigned NOT NULL,
-          slave_hosts text CHARACTER SET ascii NOT NULL,
+          source_host varchar(128) CHARACTER SET ascii NOT NULL,
+          source_port smallint(5) unsigned NOT NULL,
+          replica_sql_running tinyint(3) unsigned NOT NULL,
+          replica_io_running tinyint(3) unsigned NOT NULL,
+          primary_log_file varchar(128) CHARACTER SET ascii NOT NULL,
+          read_primary_log_pos bigint(20) unsigned NOT NULL,
+          relay_primary_log_file varchar(128) CHARACTER SET ascii NOT NULL,
+          exec_primary_log_pos bigint(20) unsigned NOT NULL,
+          seconds_behind_primary bigint(20) unsigned DEFAULT NULL,
```

Comment:
`seconds_behind_source`

Reply:
you could also use replication_lag_seconds. That's what we ended up doing in all other places besides vtorc.